sdk/storage/azure-storage-common/inc/azure/storage/common/storage_credential.hpp

```diff
@@ -62,6 +63,7 @@ namespace Azure { namespace Storage {
     friend struct Sas::BlobSasBuilder;
     friend struct Sas::ShareSasBuilder;
     friend struct Sas::DataLakeSasBuilder;
+    friend struct Sas::QueueSasBuilder;
```

Comment:
Make sure to set VCPKG config file to depend on the latest storage common package containing this new structs.

Reply:
Specifying version in vcpkg.json doesn't work very well now, and it's removed from other packages.

Reply:
We still can and should request minimum versions via `find_dependency()` and `find_component()`, search for other occurrences in our codebase - they all are versioned now.

I think that version constraints may come back as early as before the next release - https://github.com/Azure/azure-sdk-for-cpp/issues/2735 has details.

And then, having the min-versions in `find_dependency/component()` will be very useful to re-add `version>=` in the json manifests.